easybuild/easyconfigs/c/Cython/Cython-0.27.3-GCC-8.2.0-2.31.1.eb

```diff
@@ -0,0 +1,54 @@
+##
+# EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2013 The Cyprus Institute
+# Authors::   Andreas Panteli <a.panteli@cyi.ac.cy>,
+#             Thekla Loizou <t.loizou@cyi.ac.cy>,
+#             George Tsouloupas <g.tsouloupas@cyi.ac.cy>
+# License::   MIT/GPL
+#
+# Updated: Pavel Grochal (INUITS)
+##
+easyblock = 'PythonPackage'
+
+name = 'Cython'
+version = '0.27.3'
+
+homepage = 'https://cython.org/'
+description = """
+Cython is an optimising static compiler for both the Python programming
+language and the extended Cython programming language (based on Pyrex).
+"""
+docurls = [
+    "http://cython.org/#documentation",
+    "https://github.com/cython/cython",
+]
+
+toolchain = {'name': 'GCC', 'version': '8.2.0-2.31.1'}
```

Comment:
@Darkless012 It's OK to use `GCCcore` here imho (don't forget to also add `binutils` as build dep if you change that)